DevNotes.md

```diff
@@ -0,0 +1,63 @@
+# Developer Notes
+
+### Development Workflow
+
+**Tools Required**:
+
+* `just`: https://github.com/casey/just \
```

Comment:
I think we should stick to npm scripts, as they are usually the standard for node projects. If you really want to use `just` you can always install it as a dev dependency, and then provide both a justfile and npm scripts